src/index.ts

```diff
@@ -1,3 +1,10 @@
+import axios from 'axios';
+import { type } from 'os';
+
+type productId = 1;
+type included =
+  'images,items,options.values,variants.optionValues.option,groups,exclusiveGroups';
```

Comment:
This is not the correct syntax for declaring types.
type can be used inline for params